sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/util/AzureSpringIdentifier.java

```diff
@@ -92,6 +92,9 @@ private static String getVersion() {
             Properties properties = PropertiesLoaderUtils.loadProperties(
                 new ClassPathResource("azure-spring-identifier.properties"));
             version = properties.getProperty("version");
+            if (version.contains("beta")) {
```

Comment:
You should add unit tests and note why to add this logic here.

Reply:
I would recommend that we add a logic here, to define the max size a version could reach. And if a version length exceeds that MAX, and the version contains "beta" or "alpha", we then replace the "beta" with "b", and "alpha" with "a".